PlayerControls/sources/DefaultControlsViewController.swift

```diff
@@ -785,9 +786,9 @@ public final class DefaultControlsViewController: ContentControlsViewController
         
         airPlayView.props = AirPlayView.Props(
             icons: AirPlayView.Props.Icons(
-                normal: UIImage.init(named: "icon-airplay", in: Bundle(for: AirPlayView.self), compatibleWith: nil)!,
-                selected: UIImage.init(named: "icon-airplay-active", in: Bundle(for: AirPlayView.self), compatibleWith: nil)!,
-                highlighted: UIImage.init(named: "icon-airplay-active", in: Bundle(for: AirPlayView.self), compatibleWith: nil)!)
+                normal: UIImage.init(named: "icon-airplay", in: controlsBundle(for: AirPlayView.self), compatibleWith: nil)!,
```

Comment:
Can you use `createImage(named: )` from `Utils.swift`?

Reply:
`view.createImage(named: "icon-airplay")` looks not so good...